lms/resources/_js_config.py

```diff
@@ -60,6 +65,15 @@ def config(self):
             "urls": self._urls,
         }
 
+    def enable_content_item_selection_mode(self):
+        """
+        Put the JavaScript code into "content item selection" mode.
+
+        This mode shows teachers an assignment configuration UI where they can
+        choose the document to be annotated for the assignment.
+        """
+        self.config["mode"] = "content-item-selection"
```

Comment:
There's no need for an `enable_basic_lti_launch_mode()` because that's the default.

Reply:
Would this be nicer as something which accepted a value (maybe an Enum)? We only have two values right now, but any additions would cause a proliferation of `enable_<mode>_mode()` functions. A `@property` setter would allow something along the lines of:

```
config.mode = ConfigMode.CONTENT_ITEM_SELECTION
```

Reply:
In future PRs this method is actually gonna grow a large amount of stuff that needs to be set whenever we go into content item selection mode (whenever we call `js_config.enable_content_item_selection_mode()`), beyond just setting the `"mode"` string. And `enable_content_item_selection_mode()` is also gonna grow more params beyond the `mode` param that are specific to content item selection mode. So if we had an `enable_mode(mode)` method it'd have to be something like:

```python
def enable_mode(self, mode, *args, **kwargs):
    self.config["mode"] = mode

    if "mode" == "content-item-selection":
        self._enable_content_item_selection_mode(*args, **kwargs)

def _enable_content_item_selection_mode(self, ...):
    # Sets loads of things.
```

For now, I think I'll opt to leave it as it is and deal with it in the future if/when we have a proliferation of different modes, because that will be the time when we'll be able to know which code design will work best for whatever is needed to enable all those different modes.